packages/documentation/src/pages/architecture/unit-tests.mdx

```diff
@@ -0,0 +1,120 @@
+## How do unit tests work?
+
+#### Where they live
+- Tests can live anywhere in the `src` directory, but in general are located in a directory close to the code or application that they are written to test.
+- The `.unit.spec.js` file extension is used for unit tests, while `.e2e.spec.js` is used for end-to-end (E2E) tests.
+- Visual regression test config has its own directory under `src/platform/testing/visual-regression`
+- Test commands are specified in `package.json`, as well as in vets-website's `README.md`
+- Configuration files for nightwatch and sass-lint live under `config/`
```

Comment:
What do you think about re-organizing this document around the different test suites and listing the how, when, and frameworks for each?